src/timepicker/timepicker.component.ts

```diff
@@ -65,11 +73,28 @@ export class TimePicker {
 
 	@Output() valueChange: EventEmitter<string> = new EventEmitter();
 
+	writeValue(obj: any): void {
+		this.value = obj;
+	}
+	registerOnChange(fn: any): void {
+		this.onChangeHandler = fn;
+	}
+	registerOnTouched(fn: any): void {
+		this.onTouchedHandler = fn;
+	}
+	setDisabledState(isDisabled: boolean): void {
```

Comment:
You'll need to add spaces between all these methods ... like:
```suggestion

	setDisabledState(isDisabled: boolean) {
```

Additionally you can elide the `void` type, since that's implied by the empty return.